python/fnllm/fnllm/caching/file.py

```diff
@@ -55,19 +67,28 @@ async def set(
         self, key: str, value: Any, metadata: dict[str, Any] | None = None
     ) -> None:
         """Write a value into the cache."""
-        content = json.dumps(
-            {"result": value, "metadata": metadata}, indent=2, ensure_ascii=False
-        )
+        create_time = time.time()
+        content = {
+            "result": value,
+            "metadata": metadata,
+            "created": create_time,
+            "updated": create_time,
```

Comment:
Since "updated" is also set by `.get`, maybe we should call this `last_accessed`, `accessed` or something similar?

Reply:
I can rename it - updated is meant to indicate last access date